src/metabase/search/spec.clj

```diff
@@ -257,15 +258,26 @@
    (for [[search-model spec-fn] (methods spec)]
      (search-model-hooks (spec-fn search-model)))))
 
+(defn- instance->db-values
+  "Given a transformed toucan map, get back a mapping to the raw db values that we can use in a query."
+  [instance]
+  (let [xforms (#'t2.transformed/in-transforms (t2/model instance))]
```

Comment:
@camsaul Is there a better way to do this transformation without digging into toucan2 internals like this?

Reply:
Even using the slightly less convenient, but at least public, `t2.transformed/transformations` feels a bit wrong, as its still in an "implementation" namespace.

Reply:
Gonna merge as-is for now just to get things working.